### ISSUE TITLE
Several loadout items do not spawn.

### ISSUE THREAD

**Issue by IrkallaEpsilon** [Skyrat-SS13/Skyrat13#2516]
## Fix/Workaround: Hit clear loadout in the loadout screen. Reselect all items one previously had.

**Comment by Weblure** [comment]
I've had this bug for so long, thank you so much for finding a workaround.

I've noticed that a lot of people never had this bug, so I'm thinking the removal of crowbars as an option is what's breaking it for certain people (namely those that used the crowbar loadout option), especially considering how you fixed the problem. My guess is that the crowbar is still being stored in player's variables, which causes an error when it tries to spawn the items from your loadout. Clearing your loadout purges this variable of everything within it, which would be the only way to remove the crowbar from it.

 Did you have a crowbar selected in any of your characters' loadouts before it was removed?
